lib/launchers/process.js

```diff
@@ -64,11 +64,11 @@ var ProcessLauncher = function(spawn, tempDir, timer) {
 
     var errorOutput = '';
 
-    self._process.on('close', function(code) {
+    self._process.on('exit', function(code) {
       self._onProcessExit(code, errorOutput);
     });
 
-    self._process.on('error', function(err) {
+    self._process.on('karma_error', function(err) {
```

Comment:
This line does not look correct. The error event in this case is coming from the child process, indicating that spawn failed. It is different from the error event emitted by the socket, which was renamed from "error" to "karma_error".